test/micro/org/openjdk/bench/java/nio/ByteBuffers.java

```diff
@@ -923,4 +923,9 @@ public double testDirectLoopGetDouble() {
         }
         return r;
     }
+
+    @Benchmark
+    public int testHashCode() {
```

Comment:
The methods in this benchmark are named testHeapXXX and testDirectXXX so this method will need to be renamed.

Reply:
Done